apps/explorer/src/index.tsx

```diff
@@ -25,11 +27,15 @@ if (import.meta.env.PROD) {
     });
 }
 
+const adapters = [new WalletStandardAdapterProvider()];
+
 ReactDOM.createRoot(document.getElementById('root')!).render(
     <React.StrictMode>
-        <Router>
-            <App />
-        </Router>
+        <WalletProvider adapters={adapters} autoConnect={false}>
```

Comment:
I'd probably push this a little lower into the tree, which would let us effectively code split better.

Reply:
Even if I move it now, are we going to have to move it here eventually, probably once another part of the app needs it?